Minio/DataModel/PostPolicy.cs

```diff
@@ -158,6 +158,22 @@ public void SetSuccessStatusAction(string status)
         }
 
         /// <summary>
+        /// Set user specified metadata as a key/value couple.
+        /// </summary>
+        /// <param name="status">Key and Value to insert in the metadata</param>
+
+        public void SetUserSpecifiedMetadata(string key, string value)
+        {
+            if (string.IsNullOrEmpty(key))
+                throw new ArgumentException("Key is Empty");
+            if (string.IsNullOrEmpty(value))
+                throw new ArgumentException("Value is Empty");
+            string headerName = "x-amz-meta-" + key;
+            this.policies.Add(new Tuple<string, string, string>("eq", "$" + headerName, value));
+            this.formData.Add(headerName, value);
+        }
```

Comment:
The SetKey() method already allows the user to set form data, and the custom metadata can be passed in with the x-amz-meta prefix. Is there a reason why an additional method would help

Reply:
It is coming from here https://github.com/minio/minio-go/pull/839